packages/corewar-ui/package.json

```diff
@@ -33,15 +35,18 @@
     "styled-components": "^3.2.3",
     "styled-property": "^1.0.2",
     "typeface-anonymous-pro": "0.0.44",
-    "typeface-inter": "^3.11.2",
+    "typeface-inter": "^3.12.0",
     "typeface-lato": "0.0.44"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "npm run build:style && react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
     "coverage": "react-scripts test --env=jsdom --coverage",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "build:style": "tailwind build src/styles/index.css -o src/styles/tailwind.css",
+    "prebuild": "npm run build:style",
```

Comment:
Currently running `prepublish` after `bootstrap` to build. Should we be using `prebuild` instead, as well? Confused!

Reply:
In this context, the `pre-start` is to handle `npm start` locally and `pre-build` to handle a production build on a server / ci